malaria-ios/Information/EndpointType.swift

```diff
@@ -21,10 +21,14 @@ import Foundation
 /// - `Objectives`
 /// - `Goals`
 public enum EndpointType : String{
-    case BaseUrl = "http://pc-web-dev.systers.org/"
+    #if DEBUG
+    case BaseUrl = "https://systerspcweb.herokuapp.com/"
+    #else
```

Comment:
I was questioning that myself. No need. I will test the the pull request as soon as I get the chance. I only got the chance to review the code therefore, I just need to check if things are running properly.